s2-automate/s2-automate-core/src/commonTest/kotlin/s2/automate/core/engine/S2AutomateEngineImplOuterExceptionTest.kt

```
@@ -0,0 +1,417 @@
+        ): InitTransitionAppliedContext<TestState, String, TestEntity, TestEvent, S2Automate> = context
+
+        override suspend fun verifyTransition(
+            context: TransitionAppliedContext<TestState, String, TestEntity, TestEvent, S2Automate>
+        ): TransitionAppliedContext<TestState, String, TestEntity, TestEvent, S2Automate> = context
+    }
+
+    /** Guard that always passes — used for non-guard failure scenarios. */
+    private class PassthroughGuardVerifier :
+        GuardVerifier<TestState, String, TestEntity, TestEvent, S2Automate> {
+
+        override suspend fun evaluateInit(context: InitTransitionContext<S2Automate>) {}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)